tests/test_client.py

```diff
@@ -1,43 +1,64 @@
 import concurrent
-from unittest.mock import ANY, MagicMock
+import time
+from unittest.mock import ANY, MagicMock, patch
 
 from google.cloud.pubsub_v1 import PublisherClient
 
 from rele import Publisher
 
 
 class TestPublisher:
+    @classmethod
+    def setup_method(cls):
```

Comment:
We usually create a class-scoped fixture for this instead

Reply:
Done.